client/components/FailingAssertionsSummary/Table.jsx

```diff
@@ -21,7 +21,8 @@ const FailingAssertionsSummaryTable = ({
     <>
       <p>
         {failingAssertions.length} assertions failed across{' '}
-        {metrics.commandsCount} commands in {metrics.testsFailedCount} tests.
+        {failingAssertions.uniqueCommandsCount} commands in{' '}
```

Comment:
This needs some conditional copy here for single/plurals handling or it could read:

> 1 assertions failed across 1 commands in 1 tests.

Also thinking the "across" may not read well in a situation above, so examples could be:
> 1 assertion failed for 1 command in 1 test.
> 2 assertions failed for 1 command in 1 test.
> 1 assertion failed across 2 commands in 1 test.
> 1 assertion failed for 1 command in 2 tests. (this one isn't possible with the current set up so asking a follow-up below)

I don't feel strongly on the conditional "across" -> "for" though so I'm willing to go along without that particular change.

Reply:
Ah yes, thanks for catching the plurals!  I also agree with "for" replacing "across". That word made more sense when the number was for the total number of commands included in the report.

Completed in [8c2e38c](https://github.com/w3c/aria-at-app/pull/1358/commits/8c2e38cf3a72ec68cac0e765fcc8f43c8d5f8f08)

Reply:
Looks good! Thanks for addressing!